Makefile

```diff
@@ -204,7 +212,7 @@ endif
 # build docker image
 .PHONY: image/build
 image/build:
-	${DOCKER} build . -f ${DOCKERFILE} -t ${IMG}:$(IMG_VERSION)
+	${DOCKER} build . -f ${DOCKERFILE} -t ${IMG}:$(IMG_VERSION) --network host
```

Comment:
tbh I'm not sure we need this change 🤔 , further it does _not_ work with Podman desktop on my end:
![image](https://github.com/kubeflow/model-registry/assets/1699252/333474e6-b741-4ba1-8348-09e58de732d7)

Reply:
On podman Desktop on my Mac I get this error:

```
No routable interface for IPv6: IPv6 is disabled
Couldn't open network namespace /proc/4047/ns/net: Permission denied
```